### PR TITLE
Update OpenAPI spec to match v0.3.0 of the API

### PR DESCRIPTION
**Pull request by Cruikshanks**
This updates our spec and schema to match the v0.3.0 release candidate for the charging module API.

The changes are relavtibely minor

- all admin only routes are now found on the `/admin` path
- 2 new health endpoints; airbrake and database
- some new info in the view bill run summary endpoint and the filter params

### CODE REVIEW

**Comment by Cruikshanks** [comment]
Just in case it's not clear @StuAA78 you are not expected to review `openapi/versions/swagger_v0-3-0.yml` 😂 . Just the changes to the spec itself. Shout if not comfortable reviewing changes to OpenAPI spec files.

File: openapi/paths/v1/billruns/bill_run.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 get:
   operationId: ViewBillRun
-  description: "Request to view a single (billed or unbilled) bill run based on the bill run ID. Triggers generation of summary stats, which are contained in the response.Request may include optional other parameters which result in response being narrowed down to cover a subset of transactions contained within the bill run. List of individual transactions is not included in the response."
+  description: "Request to view a single (billed or unbilled) bill run based on the bill run ID. Triggers generation of summary stats, which are contained in the response.Request may include optional other parameters which result in response being narrowed down to cover a subset of transactions contained within the bill run. List of individual transactions is not included in the response.
```
Comment (StuAA78) [inline openapi/paths/v1/billruns/bill_run.yml:3]: Do we need more detail here on the summary generation? ie. that it only generates the summary if it's been cleared by adding or removing a transaction, and a holding response may be returned if this takes more than a few seconds?

**Review by StuAA78 — CHANGES_REQUESTED** [review]
One small note re: `ViewBillRun` and summary generation, other than that all good 👍

File: openapi/paths/v1/billruns/bill_run.yml (hunk 1 above)
Comment (Cruikshanks) [reply openapi/paths/v1/billruns/bill_run.yml:3]: 💯  Good point. We didn't have this when I first started documenting the API but now is a good time to capture this.

**Review by StuAA78 — APPROVED** [review]

**Pull request merged**